lib/seahorn/BvOpSem2.cc

```diff
@@ -530,7 +530,9 @@ class OpSemVisitor : public InstVisitor<OpSemVisitor>, OpSemVisitorBase {
       auto *val = CS.getArgument(2);
       Expr symVal = lookup(*val);
       if (symVal && arg0 && arg1) {
-        res = m_ctx.alu().Extract({symVal, val->getType()->getScalarSizeInBits()}, arg0->getZExtValue(), arg1->getZExtValue());
+        res =
```

Comment:
space issue

Reply:
This has been auto reformatted by the editor(vscode) using clang-format. If this is a issue with not following clang-format, I can fix it.

Reply:
ok. maybe it just looks strange in the web interface